tool/piped-base/dockertest/check-helm.sh

```diff
@@ -36,11 +36,11 @@ done
 
 declare -A versioncases
 #versioncases["helm-2.16.7"]="Client: \&version.Version{SemVer:\"v2.16.7\", GitCommit:\"5f2584fd3d35552c4af26036f0c464191287986b\", GitTreeState:\"clean\"}"
-versioncases["helm"]="version.BuildInfo{Version:\"v3.2.1\", GitCommit:\"fe51cd1e31e6a202cba7dead9552a6d418ded79a\", GitTreeState:\"clean\", GoVersion:\"go1.13.10\"}"
+versioncases["helm"]="version.BuildInfo{Version:\"v3.8.2\", GitCommit:\"6e3701edea09e5d55a8ca2aae03a68917630e91b\", GitTreeState:\"clean\", GoVersion:\"go1.17.5\"}"
 
 for h in "${!versioncases[@]}"
 do
-    got=$($h version --client)
+    got=$($h version --client --registry-config ${HELM_REGISTRY_CONFIG})
     want=${versioncases[$h]}
     if [[ ${got} == ${want} ]]; then
         echo -e "PASSED: Correct version for ${h}."
```

Comment:
@Szarny We are on the way to getting rid of Kapetanios from PipeCD so you can remove all of those docker tests.

Reply:
@nghialv Thank you for your confirmation. I checked that the correct version of Helm is installed on the local environment so I'll remove it.